### ISSUE TITLE
Support for Mobile Linux UI

### ISSUE THREAD

**Issue by n1zzo** [flxzt/rnote#67]
Hello,
thanks for creating this nice project.
I believe this could be a nice fit for linux distributions for smartphones like [postmarketOS](https://postmarketos.org/).
Right now a bit of interface tweak is needed, do you think this can be achieved with a small effort?
Here is a screenshot of the current appearance of rnote when executing on a Xiaomi Pocophone F1 with pmOS.
![20220127_10h40m23s_grim](https://user-images.githubusercontent.com/6890834/151334912-83db32f6-f5f7-4e5c-946f-473f345701aa.png)
.

**Comment by flxzt** [comment]
Is it not possible to have a smaller scale factor? Look at #54 for an improvement, but not sure if this is enough. Feedback and ideas are very welcome as I don't have a linux phone unfortunately.

**Comment by n1zzo** [comment]
After #54 things are getting a lot better.
However the minimum width is still too large for low-resolution screen like the Pine64 PinePhone.

![20220207_14h37m14s_grim](https://user-images.githubusercontent.com/6890834/152956393-688ef3b0-c707-4b01-861c-acf8e8b2ecca.png)

Can we further reduce paddings to fit this case too?

**Comment by flxzt** [comment]
A UI that works for portrait screen smartphones is out-of-scope atm. The smallest screen which is realistic to support is landscape tablet screens

**Comment by Nine-H** [comment]
I've been testing rnote on a snapdragon phone and the performance is really good. the accuracy and responsiveness of powered capacitative styluses is surprisingly good these days.

I think the solution for scaling down to a handset is to condense the overlay toolbars down to a bottom toolbar similar to the one on gnome-podcasts, something like this:

<img width="1024" height="1024" alt="Image" src="https://github.com/user-attachments/assets/51bc8d22-767a-46e0-9dd3-7aa9327b9c0c" />

in my mind the buttons across the bottom would be undo, tools, tool settings, colors/pallette, and a toggle to draw with touch input.

the reason for only retaining undo is that space is at a premium and the bar starts getting crowded quickly.

the reason the option buttons correlate to the toolbars is because the controls of each bar would need to be re laid out in a grid, there may be a way to do this automatically, it may require manual maintenance, in both cases keeping the same groupings make feature parity between both interfaces more simple to maintain. because the popovers don't have to be skinny I think settings that are submenu popovers on the toolbars (like brush styles) should be directly included in the grid layouts of each menu popover. I haven't mocked possible layouts for these popovers up yet.

the reason why I think there should be a toggle for drawing with touch input is because on a handset without an active digitiser this is one of the controls used the most, and it's clumsy to reach in the view settings popover.
